flang/tools/f18/CMakeLists.txt

```diff
@@ -69,3 +69,6 @@ if (NOT WIN32)
     FILE_PERMISSIONS OWNER_EXECUTE OWNER_READ OWNER_WRITE)
   install(PROGRAMS ${CMAKE_BINARY_DIR}/bin/flang DESTINATION bin)
 endif()
+
+# TODO Move this to a more suitable location
```

Comment:
What would be a more suitable location?

Reply:
I don't have the answer, but it definitely does not belong in "flang/tools/f18". IIUC, the goal is to replace this altogether: https://github.com/llvm/llvm-project/issues/54800

Reply:
I suppose that omp_lib.h belongs in the openmp tree because omp_lib.h is an interface to OpenMP.   I believe -- didn't look (lazy) -- that the omp.h is in the openmp tree and that it is versioned by OpenMP versions.